_infra/helm/locust/locustfiles/requests.json

```diff
@@ -58,8 +41,8 @@
         },
         {
             "method": "GET",
-            "harvest_url": {
-                "id": "surveyLongName",
+            "harvest": {
+                "ids": "surveyLongName",
```

Comment:
This should be "id"

Reply:
That was me going refactoring mad. Reverted back.